src/components/Snackbar/SnackbarContext.tsx

```diff
@@ -71,6 +71,15 @@ const SnackbarContext = React.createContext<SnackbarContextValue>({
   updateSnackbar: () => '',
 });
 
+const positionStyles = {
```

Comment:
nit: we should type that using `keyof ControlledAlertProps['position']: Record<string, any>` or smth similar

Reply:
@eliskaachee would you agree here? It's a small nitpick, but I feel it would help :)

Reply:
Updated :)